v201710/campaign_management_test.go

```diff
@@ -0,0 +1 @@
+package v201710
```

Comment:
What's this file for?

Reply:
I have no idea 🤣  musta been a test someone was starting and never finished.